intermediate_source/torch_compile_tutorial.py

```diff
@@ -175,7 +177,7 @@ def evaluate(mod, inp):
 ######################################################################
 # And indeed, we can see that running our model with ``torch.compile``
 # results in a significant speedup. On an NVIDIA A100 GPU, we observe a
-# 2.3x speedup. Speedup mainly comes from reducing Python overhead and
+# ~1.5x speedup. Speedup mainly comes from reducing Python overhead and
```

Comment:
Not sure how I feel about putting these numbers in a tutorial, you'll have to come back and update them. 

So either
1. Remove them
2. Timestamp in the doc when you last ran the benchmarks